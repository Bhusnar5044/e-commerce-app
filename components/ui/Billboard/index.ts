export { Billboard as default } from './Billboard';
