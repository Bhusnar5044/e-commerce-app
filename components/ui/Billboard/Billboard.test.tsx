import { render } from '@test-utils';
import { Billboard } from './Billboard';

describe('Billboard', () => {
    it('should render properly', () => {
        const { container } = render(<Billboard data={{ id: '', label: 'dummy', imageUrl: '' }} />);
        expect(container).toMatchSnapshot();
    });
});
