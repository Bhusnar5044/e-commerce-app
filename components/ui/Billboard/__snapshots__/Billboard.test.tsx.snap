// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Billboard should render properly 1`] = `
<div>
  <div
    class="p-4 sm:p-6 lg:p-8 rounded-xl overflow-hidden"
  >
    <div
      class="rounded-xl relative aspect-square md:aspect-[2.4/1] overflow-hidden bg-cover"
      style="background-image: url();"
    >
      <div
        class="h-full w-full flex flex-col justify-center items-center text-center gap-y-8"
      >
        <div
          class="font-bold text-3xl sm:text-5xl lg:text-6xl sm:max-w-xl max-w-xs"
        >
          dummy
        </div>
      </div>
    </div>
  </div>
</div>
`;
